assets/js/modules/pagespeed-insights/components/DashboardPageSpeedCTA.js

```diff
@@ -6,23 +24,28 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import {
 	activateOrDeactivateModule,
 	getReAuthURL,
 	showErrorNotification,
-	getModulesData,
 } from '../../../util';
 import CTA from '../../../components/notifications/cta';
 import data from '../../../components/data';
 import GenericError from '../../../components/notifications/generic-error';
+import { STORE_NAME as MODULES_STORE } from '../../../googlesitekit/modules/datastore/constants';
+import { STORE_NAME as USER_STORE } from '../../../googlesitekit/datastore/user/constants';
+const { useSelect } = Data;
 
-const PageSpeedInsightsCTA = () => {
-	const {
-		active,
-		setupComplete,
-	} = getModulesData()[ 'pagespeed-insights' ];
+function DashboardPageSpeedCTA() {
+	const pagespeedInsightsModule = useSelect( ( select ) => select( MODULES_STORE ).getModule( 'pagespeed-insights' ) );
+	const canManageOptions = useSelect( ( select ) => select( USER_STORE ).hasCapability( 'googlesitekit_manage_options' ) );
```

Comment:
We should probably add the available Site Kit capabilities as constants to the `core/user` store so that they can be referenced. While this duplicates that list between PHP and JS, it's still cleaner than accessing via hard-coded string in components.

Reply:
Ok, updated. However I have had to rename `PERMISSION_SCOPE_ERROR_CODE` into `ERROR_MISSING_REQUIRED_SCOPE` to escape confusions between error codes and permission names.